config/config.exs

```diff
@@ -1,3 +1,5 @@
 use Mix.Config
 
+config :ecto, json_library: Jason
```

Comment:
This should be part of https://github.com/parody/chronik/issues/38
Anyway I would put something like:
```
config :chronik, Chronik.Store.Adapters.Ecto
  json_library: Jason
```

Reply:
I would add also `dump_to_json: true`

Reply:
@arpunk Could you point this PR to `develop`. We should leave `master` for releases only.